### DISCUSSION TITLE
Can we trigger scenario programmatically?

### DISCUSSION THREAD

**Discussion by irynahryshanovich** [getgauge/gauge#2089] [General]
Hello,

I have scenario A and scenario B. Scenario B is dependent on Scenario A
If scenario A was not executed before B, can I trigger it somehow programmatically?

Thank you

**Comment by sriv** [comment]
Hi, This is not possible currently. There is an open issue (#1991 ) to allow more control over the order of scenarios.

**Reply by irynahryshanovich** [reply]
thank you! It will work for us, looking forward to seeing the implementation
